tests/yam/migration/migration_unattended.pm

```diff
@@ -0,0 +1,44 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Migration activation then reboot to perform migration.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use grub_utils 'grub_test';
+use migration 'disable_installation_repos';
+use power_action_utils 'power_action';
+use utils 'zypper_call';
+
+sub run {
+    my $self = shift;
+
+    #my $u = get_var('SCC_PROXY_URL');
+    #enter_cmd "echo 'url: $u' > /etc/SUSEConnect";
+
+    disable_installation_repos;
+    enter_cmd "echo 'url: " . get_var('SCC_URL') . " > /etc/SUSEConnect'";
+
+    select_console('root-console');
+
+    my $repo_home = "http://download.suse.de/ibs/home:/fcrozat:/SLES16/SLE_\$releasever";
+    my $repo_images = 'http://download.suse.de/ibs/home:/fcrozat:/SLES16/images/';
+    zypper_call("ar -p 90 '$repo_home' home_sles16");
+    zypper_call("ar -p 90 $repo_images home_images");
+
+    # install the migration image and active it
+    zypper_call("--gpg-auto-import-keys -n in suse-migration-sle16-activation");
+
+    power_action('reboot', keepconsole => 1, first_reboot => 1);
+
+    record_info 'Handle GRUB';
+    grub_test();
+    $self->wait_boot(bootloader_time => 400, textmode => 1, ready_time => 600);
```

Comment:
This is a normal boot in sle 16, this part doesn't belong to this test module.

Additionally we could check with some generic needle the blue screen of the migration, what I don't understand from your verification run why it appears so little time.

Reply:
updated code and VR.